tests/tacacs/test_accounting.py

```diff
@@ -122,6 +123,21 @@ def check_local_no_other_user_log(duthost, tacacs_creds):
     pytest_assert(len(logs) == 0, "Expected to find no accounting logs but found: {}".format(logs))
 
 
+def check_server_received(ptfhost, data):
+    """
+        Check if tacacs server received the data.
+    """
+    hex = binascii.hexlify(data.encode('ascii'))
+    hex_string = hex.decode()
+
+    # Extract received data from tac_plus.log, then use grep to check if the received data contains hex_string
+    sed_command = "sed -n 's/.*-> 0x\(..\).*/\\1/p'  /var/log/tac_plus.log | sed ':a; N; $!ba; s/\\n//g' | grep '{0}'".format(hex_string) # noqa W605 E501
```

Comment:
Fixed, add detail in comments:
            1. tac_plus server start with '-d 2058' parameter to log received data in following format in tac_plus.log:
 
                    Thu Mar  9 06:26:16 2023 [75483]: data[140] = 0xf8, xor'ed with hash[12] = 0xab -> 0x53
                    Thu Mar  9 06:26:16 2023 [75483]: data[141] = 0x8d, xor'ed with hash[13] = 0xc2 -> 0x4f
                In above log, the 'data[140] = 0xf8' is received data.

            2. Following sed command will extract the received data from tac_plus.log:
                    sed -n 's/.*-> 0x\(..\).*/\\1/p'  /var/log/tac_plus.log

                   So the result will be:
                       f8
                       8d

            3. Following set command will join all received data to hex string:
                    sed ':a; N; $!ba; s/\\n//g'

                   So the result will be:
                       f88d.....

            4. Then the grep command will check if the received hex data containes expected hex string.
                    grep '{0}'".format(hex_string)

    In our case, because we send remote address to TACACS server, so:
             1. we convert remote address to hex string:
                        "10.20.0.1"  =>  31302E32302E302E31
             2. We check if the received hex data contains "31302E32302E302E31", if it contains, the UT passed.